projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -1056,3 +1328,248 @@ export function checkout(): void {
+ * @returns {Chainable<JQuery<HTMLElement>>} - Corresponding bundle item
+ */
+function findBundleItem(cartItemIndex: number): Chainable<JQuery<HTMLElement>> {
+  return cy
+    .get('cx-cart-item-list .cx-item-list-row')
+    .eq(cartItemIndex)
+    .find('cx-configurator-cart-entry-bundle-info');
+}
+
+/**
+ * Verifies the name of bundle item.
+ *
+ * @param {number} cartItemIndex - Index of cart item
+ * @param {number} bundleItemIndex - Index of bundle item
+ * @param {string} name - Expected name of bundle item
+ */
+export function checkBundleItemName(
+  cartItemIndex: number,
+  bundleItemIndex: number,
+  name: string
+) {
+  findBundleItem(cartItemIndex).within(() => {
+    cy.get('.cx-item-info')
+      .eq(bundleItemIndex)
+      .within(() => {
+        cy.get('.cx-item-name').should('contain', name);
+      });
+  });
+}
+
+/**
+ * Verifies the price of bundle item.
+ *
+ * @param {number} cartItemIndex - Index of cart item
+ * @param {number} bundleItemIndex - Index of bundle item
+ * @param {string} price - Expected price of bundle item
+ */
+export function checkBundleItemPrice(
+  cartItemIndex: number,
+  bundleItemIndex: number,
+  price: string
+) {
+  findBundleItem(cartItemIndex).within(() => {
+    if (price) {
+      cy.get('.cx-item-info')
+        .eq(bundleItemIndex)
+        .within(() => {
+          cy.get('.cx-item-price .cx-item').should('contain', price);
+        });
+    }
+  });
+}
+
+/**
+ * Verifies the quantity of bundle item.
+ *
+ * @param {number} cartItemIndex - Index of cart item
+ * @param {number} bundleInfoIndex - Index of bundle item
+ * @param {string} quantity - Expected quantity of bundle item
+ */
+export function checkBundleItemQuantity(
+  cartItemIndex: number,
+  bundleInfoIndex: number,
+  quantity: string
+) {
+  findBundleItem(cartItemIndex).within(() => {
+    if (quantity) {
+      cy.get('.cx-item-info')
+        .eq(bundleInfoIndex)
+        .within(() => {
+          cy.get('.cx-item-quantity .cx-item').should('contain', quantity);
+        });
+    }
+  });
+}
+
+/**
+ * Toggle bundle items via 'show' or 'hide' link
+ *
+ * @param {string} linkName - Name of the toggled link
+ */
+function toggleBundleItems(linkName: string) {
+  cy.get('.cx-toggle-hide-items')
+    .should('contain', linkName)
+    .click()
+    .then(() => {
+      let expectedLinkName = 'hide';
+      if (linkName !== 'show') {
+        expectedLinkName = linkName;
+      }
+      cy.get('.cx-toggle-hide-items').should('contain', expectedLinkName);
+    });
+}
+
+/**
+ * Verifies the amount of bundle items for a certain cart item.
+ *
+ * @param {number} cartItemIndex - Index of cart item
+ * @param {number} itemsAmount - Expected amount of bundle items
+ */
+export function checkAmountOfBundleItems(
+  cartItemIndex: number,
+  itemsAmount: number
+) {
+  findBundleItem(cartItemIndex).within(() => {
+    cy.get('.cx-number-items').should('contain', itemsAmount);
+    toggleBundleItems('show');
+  });
+}
```

Comment:
Didn't you find 1,5k lines too much for one file? 😉 
For better maintainability, you might want to create a ticket to split it into more files.

Reply:
Created #12103